apps/devportal/data/markdown/pages/learn/faq/xm-personalize/development.md

```diff
@@ -6,36 +6,36 @@ hasInPageNav: false
 cdpTags: ['xm', 'personalize']
 ---
 
-**What technology stack do we need to build sites in XP2XMP?**
+### What technology stack do we need to build sites in XP2XMP?
 
 The Sitecore Experience Manager component of XP2XMP is run on a Microsoft technology stack, including Windows Server, SQL Server, .NET, and ASP.NET.
 
 Headless development of sites with Sitecore Experience Manager supports two options: (a) Node.js and (b) .NET Core. With the Node.js stack, the components can be delivered using Next.js, React, Angular, or Vue.js SDKs. Sitecore recommends using Next.js as the Jamstack framework because of its flexibility in supporting Static Site Generation (SSG), Incremental Static Regeneration (ISR), and Server-Side Rendering (SSR). Additionally, Next.js does not require customers to run a separate rendering host, which reduces operational complexity and overhead.
 
-**Can we use XP2XMP with SSR for JSS headless implementations?**
+### Can we use XP2XMP with SSR for JSS headless implementations?
 
 When using Sitecore Experience Manager from the XP2XMP bundle with the Sitecore JavaScript Rendering SDK (JSS), developers can use Server-Side Rendering (SSR) with their Next.js, React, Angular, or Vue.js applications. React, Angular, and Vue.js require customers to host an [SSR proxy](https://doc.sitecore.com/xp/en/developers/hd/190/sitecore-headless-development/server-side-render-jss-apps-headlessly-using-the-jss-proxy.html) server themselves to support SSR with JSS. Next.js supports SSR without requiring a separate SSR proxy server as this capability is supported through Next.js directly. This is one of the reasons why Sitecore recommends using Next.js as the Jamstack framework for client applications when implementing a headless solution.
 
-**We just launched our XM site on 10.1 using the best practices of SXA. Can we port that site over to XP2XMP so we can take advantage of Personalize and Send that we did not purchase with our XM License?**
+### We just launched our XM site on 10.1 using the best practices of SXA. Can we port that site over to XP2XMP so we can take advantage of Personalize and Send that we did not purchase with our XM License?

```

Comment:
Mentions of Send in a XM + Personalize document, probably doesn't make sense?

Reply:
Addressed while working on the other Send references.